transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/Message.kt

```diff
@@ -147,8 +147,9 @@ data class Message(
     data class Card(
         val title: String,
         val description: String,
-        val imageUrl: String,
-        val actions: List<Action>
+        val imageUrl: String? = null,
+        val actions: List<Action>,
+        val defaultAction: Action? = null,
```

Comment:
is defaultAction nullable? For me it sounds like the whole purpose of `defaultAction` is to be a backup in case actions are missing or if it is not specified, but TBH I do not remember schema of this object, so I might be wrong here.